SqrlForNet/SqrlAuthenticationOptions.cs

```diff
@@ -83,6 +88,8 @@ public class SqrlAuthenticationOptions : RemoteAuthenticationOptions
 
         public Action<string> HardlockReceived;
 
+        //public Func<HttpContext, AskMessage> ReturnAskMessage;
```

Comment:
This is not part of this branch but is also due to the work starting in the wrong branch so let it slid and it is a commented out line